pom.xml

```diff
@@ -714,23 +780,25 @@
 					<artifactId>cobertura-maven-plugin</artifactId>
 					<version>2.7</version>
 					<configuration>
+						<aggregate>true</aggregate>
 						<check>
 							<branchRate>0</branchRate>
+							<haltOnFailure>${verify.haltOnFailure}</haltOnFailure>
 							<lineRate>0</lineRate>
 							<packageBranchRate>0</packageBranchRate>
 							<packageLineRate>0</packageLineRate>
 							<totalBranchRate>${verify.totalBranchRate}</totalBranchRate>
 							<totalLineRate>${verify.totalLineRate}</totalLineRate>
 						</check>
+						<instrumentation>
+							<excludes>
+								<exclude>**/Test*.java</exclude>
```

Comment:
Why do we need those excludes ? There might be a good reason, in which case an inline comment would be welcomed.